utils/git_utils.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
```

Comment:
A minor suggestion re documenting this file - for the moment there's no need to document the file as both functions are documented and the file is tiny. At some point if/when the file contains more code it'll be good to document the file to summarise why it's needed, as well as the individual functions.

Reply:
- Added docstrings for the `utils/git_utils.py` file and its functions 
- pushed the code